js/modules/WebAPIProvider/VocabularyProvider.js

```diff
@@ -118,7 +124,7 @@ define(function (require, exports) {
 		return deferred.promise();
 	}
 
-	function getDomains() {
+	function getDomainPromise() {
```

Comment:
Just need to revert this rename and just have 'getDomains' exposed int he api below.